antlr/src/main/antlr4/org/apache/iotdb/db/qp/sql/IoTDBSqlParser.g4

```diff
@@ -447,23 +447,36 @@ showConfigNodes
 
 // ---- Get Region Id
 getRegionId
-    : SHOW (DATA|SCHEMA) REGIONID OF path=prefixPath WHERE (SERIESSLOTID operator_eq
-        seriesSlot=INTEGER_LITERAL|DEVICEID operator_eq deviceId=prefixPath) (OPERATOR_AND (TIMESLOTID operator_eq timeSlot=INTEGER_LITERAL|
-        TIMESTAMP operator_eq timeStamp=INTEGER_LITERAL))?
+    : SHOW (DATA|SCHEMA) REGIONID WHERE (DATABASE operator_eq database=prefixPath
+        |(DEVICE|DEVICEID) operator_eq device=prefixPath)
```

Comment:
```suggestion
        | DEVICE operator_eq device=prefixPath)
```
"DEVICEID" will be strange because we only accept input device=prefixPath

Reply:
need to consider compatibility with previous versions? I can remove that

Reply:
I think we don't need to consider the compatibility with our previous incomplete version